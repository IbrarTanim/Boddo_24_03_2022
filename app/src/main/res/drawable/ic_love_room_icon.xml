<vector xmlns:android="http://schemas.android.com/apk/res/android"
    android:width="48dp"
    android:height="48dp"
    android:viewportWidth="48"
    android:viewportHeight="48">
    <path
        android:fillColor="#F05050"
        android:pathData="M30.6,4.9c-4,0.6 -7.2,3 -9.1,6.3C18.8,8.6 15,7.3 11,7.9C4.1,8.9 -0.6,15.4 0.5,22.3c2.4,15.6 26,20.9 26,20.9S47.4,31 45,15.4C43.9,8.5 37.5,3.8 30.6,4.9z" />
    <path
        android:fillColor="#EF3D3C"
        android:pathData="M45,15.4C43.9,8.5 37.5,3.8 30.6,4.9c-0.2,0 -0.4,0.1 -0.7,0.1c4.9,1.1 8.9,5.1 9.9,10.4c2.2,11.5 -6.4,21.7 -12.2,27.1C32,39.6 47.1,28.8 45,15.4z" />
    <path
        android:fillColor="#FACBD7"
        android:pathData="M40.7,18.2c-2.9,-0.8 -5.9,-0.1 -8.1,1.6c-1.1,-2.6 -3.3,-4.6 -6.3,-5.4c-5,-1.3 -10.2,1.7 -11.5,6.7c-3,11.4 11.6,22 11.6,22s17.9,-2.1 20.9,-13.5C48.7,24.6 45.7,19.5 40.7,18.2z" />
    <path
        android:fillColor="#F7B1C9"
        android:pathData="M40.7,18.2c-1.8,-0.5 -3.6,-0.4 -5.2,0.1c4.8,1.5 7.6,6.5 6.3,11.3c-1.9,7.3 -9.8,11.4 -15.3,13.5c0,0 0,0 0,0s17.9,-2.1 20.9,-13.5C48.7,24.6 45.7,19.5 40.7,18.2z" />
</vector>
