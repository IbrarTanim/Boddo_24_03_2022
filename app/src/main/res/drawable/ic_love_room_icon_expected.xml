<vector xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:aapt="http://schemas.android.com/aapt"
    android:width="48dp"
    android:height="48dp"
    android:viewportWidth="48"
    android:viewportHeight="48">
  <path
      android:pathData="M30.6,4.9c-4,0.6 -7.2,3 -9.1,6.3C18.8,8.6 15,7.3 11,7.9c-6.9,1 -11.6,7.5 -10.5,14.4c2.4,15.6 26,20.9 26,20.9S47.4,31 45,15.4C43.9,8.5 37.5,3.8 30.6,4.9z">
    <aapt:attr name="android:fillColor">
      <gradient 
          android:startY="23.9702"
          android:startX="0.3403"
          android:endY="23.9702"
          android:endX="45.1919"
          android:type="linear">
        <item android:offset="0" android:color="#FFF15839"/>
        <item android:offset="1" android:color="#FFDD2677"/>
      </gradient>
    </aapt:attr>
  </path>
  <path
      android:pathData="M45,15.4C43.9,8.5 37.5,3.8 30.6,4.9c-0.2,0 -0.4,0.1 -0.7,0.1c4.9,1.1 8.9,5.1 9.9,10.4c2.2,11.5 -6.4,21.7 -12.2,27.1C32,39.6 47.1,28.8 45,15.4z"
      android:fillColor="#EF3D3C"/>
  <path
      android:pathData="M40.7,18.2c-2.9,-0.8 -5.9,-0.1 -8.1,1.6c-1.1,-2.6 -3.3,-4.6 -6.3,-5.4c-5,-1.3 -10.2,1.7 -11.5,6.7c-3,11.4 11.6,22 11.6,22S44.3,41 47.3,29.6C48.7,24.6 45.7,19.5 40.7,18.2z"
      android:fillColor="#FACBD7"/>
  <path
      android:pathData="M40.7,18.2c-1.8,-0.5 -3.6,-0.4 -5.2,0.1c4.8,1.5 7.6,6.5 6.3,11.3C39.9,36.9 32,41 26.5,43.1l0,0c0,0 17.9,-2.1 20.9,-13.5C48.7,24.6 45.7,19.5 40.7,18.2z"
      android:fillColor="#F7B1C9"/>
</vector>
